cmd/oceantv/broadcast_machine.go

```diff
@@ -209,6 +218,22 @@ func (sm *broadcastStateMachine) handleTimeEvent(event timeEvent) {
 		if withTimeout.timedOut(event.Time) {
 			onFailureClosure(sm.ctx, sm.ctx.cfg)(errors.New("direct starting timed out"))
 		}
+	case *vidforwardPermanentStarting:
+		withTimeout := sm.currentState.(stateWithTimeout)
+		if withTimeout.timedOut(event.Time) {
+			onFailureClosure(sm.ctx, sm.ctx.cfg)(errors.New("permanent starting timed out"))
+		}
+	case *vidforwardPermanentTransitionSlateToLive:
+		withTimeout := sm.currentState.(stateWithTimeout)
+		if withTimeout.timedOut(event.Time) {
+			notifier.Send(
+				context.Background(),
+				sm.ctx.cfg.SKey,
+				"health",
+				"transition from slate to live timed out, transitioning to failure slate state",
```

Comment:
It was mentioned in the PR comment that this would notify that the hardware was probably at fault, but doesn't seem to say that here?

Reply:
Yeah I meant more implicitly.